source/core/material/warp.cpp

```diff
@@ -59,6 +59,13 @@ namespace pov
 
 const DBL COORDINATE_LIMIT = 1.0e17;
 
+enum BlackHoleType
+{
+    kBlackHoleSpherical_Original=0,
+    kBlackHoleSpherical_Clamped,
+    kBlackHoleSpherical_WhatMightBeUseful
```

Comment:
Yes, I can remove those. I have an appointment this morning so the update will come later today.